go/vt/wrangler/materializer_test.go

```diff
@@ -2707,7 +2707,7 @@ func TestStripConstraints(t *testing.T) {
 	}
 }
 
-func TestMaterializerManyToManySomeUnreachable(t *testing.T) {
+func TestMaterializerSourceShardSelection(t *testing.T) {
```

Comment:
Not sure where the equivalent tests is for `go/vt/vtctl/workflow/materializer.go`.

Reply:
Thanks @mattlord returning to draft while I get tests added there